unit_tests/provisioner/test_azure_provisioner.py

```diff
@@ -0,0 +1,105 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
let's disable the test since it creates real resources and re-enable when the mock PR will be ready